package.json

```diff
@@ -6,6 +6,12 @@
   "author": "Eric Liu (https://github.com/metonym)",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.js"
```

Comment:
I believe `types` must also be defined (e.g., `"types": "./index.d.ts"`)

Reply:
OK | added